static/src/wur/variables.scss

```diff
@@ -32,9 +32,13 @@ $link-color: $black;
 $list-group-active-color: $black;
 $list-group-disabled-color: $black;
 
-// Bootstrap bage
+// Bootstrap badge
 $badge-font-size: .65em;
 
+// Group manager table
+$table-active-color: $black;
+$table-active-bg: $primary;
+
```

Comment:
For the WUR theme, I still made the text a dark color on the green background of the active item, as I noticed a previous commit tried to preserve that state https://github.com/UtrechtUniversity/yoda-portal/commit/ba8c8460930383f32434ef91421b14b9acbb2d95